.gitignore

```diff
@@ -5,6 +5,9 @@ build/
 !**/src/test/**/build/
 **/cacheDir/
 
+### Maven
+**/target
```

Comment:
we can get rid of gradle gitignores now, yea?

Reply:
We could, but since its not harmful I'd rather leave it here... Also especially since existing devs when pull this new version might have their gradle folders and commit them by accident (not biggie, just a nit)

Reply:
@jiyoontbd and I agree w/ Leo, let's leave